src/uct/ib/ud/base/ud_iface.h

```diff
@@ -123,8 +123,11 @@ struct uct_ud_iface {
     ucs_ptr_array_t       eps;
     uct_ud_iface_peer_t  *peers[UCT_UD_HASH_SIZE];
     struct {
-        int               timer_id;
-        ucs_twheel_t      slow_timer;
+        int                       timer_id;
```

Comment:
can swap order of "timer_id" and "slow_timer" to give chance for compiler to reduce structure size